### PR TITLE
Fix #12: Make the capture IDL attribute a DOMString

### LINKED ISSUE #12: Define clearly how the capture IDL attribute is reflected
https://w3c.github.io/html-media-capture/#the-capture-attribute says "The capture IDL attribute must reflect the respective content attribute of the same name." and links to https://html.spec.whatwg.org/multipage/infrastructure.html#reflect

However, the bit in HTML that defined how to reflect an enum was removed in https://github.com/whatwg/html/pull/2585. Probably the solution is to make the IDL attribute a DOMString, and handling it similarly to for example https://html.spec.whatwg.org/multipage/embedded-content.html#attr-track-kind

@annevk @domenic

### PR DESCRIPTION
**Pull request by anssiko**
Fixes #12.

PTAL @foolip @domenic.

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/html-media-capture/issue-12-capture-domstring.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/html-media-capture/80977db...b89f26a.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -195,39 +200,38 @@
...
         };
       </pre>
       <div>
         <p>
-          The <dfn data-dfn-for="CaptureFacingMode">CaptureFacingMode</dfn>
-          enumeration is used to express the <a>preferred facing mode</a>. The
-          semantics of its keywords mirror the similarly named keywords defined
-          in <a><code>VideoFacingModeEnum</code></a>.
+          The <a>capture</a> attribute is an <a>enumerated attribute</a> that
+          specifies the <a>preferred facing mode</a> for the <a>media capture
```
Comment (domenic) [inline index.html]: Maybe "whose state specifies"?

File: index.html [hunk 2]
```diff
@@ -195,39 +200,38 @@
...
         </p>
-        <p class="note">
-          If the user agent is unable to support the <a>preferred facing
-          mode</a>, it can fall back to the implementation-specific default
-          facing mode.
+        <p>
+          The attribute's keywords are <dfn><code>user</code></dfn> and
+          <dfn><code>environment</code></dfn>, which map to the respective
+          states <var>user</var> and <var>environment</var>. The semantics of
+          the keywords <a>user</a> and <a>environment</a> mirror the similarly
```
Comment (domenic) [inline index.html]: You should probably be talking about the semantics of the states?

File: index.html [hunk 3]
```diff
@@ -195,39 +200,38 @@
...
+          named keywords defined in <a><code>VideoFacingModeEnum</code></a>.
```
Comment (domenic) [inline index.html]: enumeration values, not keywords, in this case

**Review by domenic — APPROVED** [review]
LGTM with some minor suggestions. Thanks for being so responsive to the HTML churn here.

It would be ideal if somewhere the processing model was specified so that something actually used these three states, but I guess that was a preexisting issue. "whose state specifies the preferred facing mode" is reasonable for now.

**Comment by anssiko** [comment]
@domenic thanks for your swift review! I addressed your comments.

**Pull request merged**